SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+## Supported Versions
+Legacy-compatibility releases `v1.x`:
+- :hourglass_flowing_sand: Web3.js currently undergoes a quick release cycle with regular minor and patch releases.
+- :globe_with_meridians: We encourage using the [most recently released version](https://github.com/ChainSafe/lodestar/releases/latest) from the `v1.x` release track before reporting an issue.
+
+Future rewrites and feature tracks `v2.x`, `v3.x`, and  `v4.x`:
+- :hourglass: Web3.js currently undergoes a complete refactoring and rewrite. At this time it is not recommended to use any of these release tracks, yet.
```

Comment:
Does the term rewrite only valid for `4.x` branch?

Reply:
yeah, that's normal when you submit PRs before first coffee 😆

Reply:
> Does the term rewrite only valid for `4.x` branch?

That's generally how we've been using that term. rewrite = 4.x